app/assets/stylesheets/govuk_publishing_components/_all_components.scss

```diff
@@ -91,6 +91,7 @@
 @import "components/summary-list";
 @import "components/tabs";
 @import "components/table";
+@import "components/tag";
```

Comment:
I don't know why, but tags have a maximum width of 160px set, which is noticeable if you display a tag in Firefox and use 'zoom text only' (or just increase the default font size in your browser). Fixable with `max-width: none` in the gem's tag scss.

<img width="527" height="218" alt="Screenshot 2025-07-15 at 11 29 38" src="https://github.com/user-attachments/assets/b4c439fe-3d54-405f-8510-e9a4a798a13b" />